distribution/docs/src/main/resources/_contents/_core-concepts/federation-intro-contents.adoc

```diff
@@ -4,3 +4,13 @@ By default, the ${branding} is able to federate using http://www.opensearch.org/
 The minimum configuration necessary to configure those federations is to supply a query address.
 
 Federation enables constructing dynamic networks of data sources that can be queried individually, or aggregated into specific configuration to enable a wider range of accessibility for data and data products.
+
+Federation provides the capability to extend the ${branding} enterprise to include <<_sources,Remote Sources>>, which may include other instances of ${branding}. 
+The Catalog handles all aspects of federated queries as they are sent to the Catalog Provider and Remote Sources, as they are processed, and as the query results are returned.
+Queries can be scoped to include only the local Catalog Provider (and any Connected Sources), only specific Federated Sources, or the
+entire enterprise (which includes all local and Remote Sources).
```

Comment:
Might be because I am not familiar with asciidoc, but what's with this weird line break right here?